Detectors/TPC/reconstruction/src/KrBoxClusterFinder.cxx

```diff
@@ -112,7 +90,59 @@ void KrBoxClusterFinder::fillADCValue(int cru, int rowInSector, int padInRow, in
     adcValue /= correctionFactor;
   }
 
-  mMapOfAllDigits[timeBin][rowInSector][corPad] = adcValue;
+  mTempTimeSlice[rowInSector][corPad] = adcValue;
+}
+
+void KrBoxClusterFinder::fillTempTimeSlice(std::vector<o2::tpc::Digit>& eventSector, const int timeSlice)
```

Comment:
Can't one simplify the filling of the slice and get rid of the `mTempTimeSlice`, remove
```
  void addTimeSliceToMap();
  void clearTempTimeSlice();
```

And call this function
```suggestion
void KrBoxClusterFinder::addTimeSlice(std::vector<o2::tpc::Digit>& eventSector, const int timeSlice)
```
together with the canges below.

This will avoid copying and unnecessary complexity.